pkg/controller/rbac.go

```diff
@@ -65,6 +67,81 @@ func (c *StashController) ensureSidecarRoleBindingDeleted(resource metav1.Object
 		Delete(c.getSidecarRoleBindingName(resource.Name), &metav1.DeleteOptions{})
 }
 
+func (c *StashController) ensureCronJobRBAC(resource *core.ObjectReference) error {
+	if err := c.ensureCronJobRole(resource); err != nil {
+		return err
+
+	}
+	if err := c.ensureCronJobRoleBinding(resource); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (c *StashController) ensureCronJobRole(resource *core.ObjectReference) error {
+	meta := metav1.ObjectMeta{
+		Name:      CronJobRole,
+		Namespace: resource.Namespace,
+	}
+	_, _, err := rbac_util.CreateOrPatchRole(c.kubeClient, meta, func(in *rbac.Role) *rbac.Role {
+		core_util.EnsureOwnerReference(&in.ObjectMeta, resource)
+		if in.Labels == nil {
+			in.Labels = map[string]string{}
+		}
+		in.Labels["app"] = "stash"
+		in.Rules = []rbac.PolicyRule{
+			{
+				APIGroups: []string{api_v1beta1.SchemeGroupVersion.Group},
+				Resources: []string{"backupsessions"},
+				Verbs:     []string{"create", "patch", "get", "list"},
+			},
+		}
+		return in
+	})
+	return err
+}
+
+func (c *StashController) ensureCronJobRoleBinding(resource *core.ObjectReference) error {
```

Comment:
Set `BackupConfiguration` as owner of this RoleBinding.

Reply:
resolved